### PR TITLE
[202012] Apply separated DSCP_TO_TC_MAP and TC_TO_QUEUE_MAP on dualtor

### PR DESCRIPTION
**Pull request by bingwang-ms**
#### Why I did it
This PR is to backport #12730 into `202012` branch.
The PR is to apply separated `DSCP_TO_TC_MAP` and `TC_TO_QUEUE_MAP` to uplink ports on dualtor.
The traffic with `DSCP 2` and `DSCP 6` from T1 is treated as lossless traffic.
```
DSCP    TC    Queue
2      2     2
6      6     6
```
Traffic with `DSCP 2` or `DSCP 6` from downlink is still treated as lossy traffic as before.

#### How I did it
Define `DSCP_TO_TC_MAP|AZURE_UPLINK` and `TC_TO_QUEUE_MAP|AZURE_UPLINK`.

#### How to verify it
1. Verified by UT
2. Verified by coping the new template to a testbed, and rendering a `config_db.json`

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

#### Description for the changelog

Apply separated `DSCP_TO_TC_MAP` and `TC_TO_QUEUE_MAP` to uplink ports on dualtor

### CODE REVIEW

File: device/common/profiles/th2/7260/RDMA-CENTRIC/qos.json.j2 [hunk 1]
```diff
@@ -218,6 +218,72 @@
             "62": "1",
             "63": "1"
         },
+        "AZURE_UPLINK": {
```
Comment (neethajohn) [inline device/common/profiles/th2/7260/RDMA-CENTRIC/qos.json.j2:221]: different flags missing

File: device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8/BALANCED/qos.json.j2 [hunk 2]
```diff
@@ -1,4 +1,6 @@
 {% if 'subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR' %}
+{% set different_dscp_to_tc_map = true %}
```
Comment (neethajohn) [inline device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8/BALANCED/qos.json.j2:2]: need same set of changes for C32 sku as well

Comment (bingwang-ms) [reply device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8/BALANCED/qos.json.j2:2]: The [Arista-7050CX3-32S-C32](https://github.com/sonic-net/sonic-buildimage/tree/202012/device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32)/[BALANCED](https://github.com/sonic-net/sonic-buildimage/tree/202012/device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/BALANCED)/qos.json.j2 is linked to [Arista-7050CX3-32S-D48C8/BALANCED/qos.json.j2](https://github.com/sonic-net/sonic-buildimage/pull/12792/files/0422dc61d894592045f7a886a70c450c74791b1c#diff-2fe712257b431ce432d1903123a08e1cf0dc304597369e0a44acbb60c196888a)

File: device/common/profiles/th2/7260/RDMA-CENTRIC/qos.json.j2 (hunk 1 above)
Comment (bingwang-ms) [reply device/common/profiles/th2/7260/RDMA-CENTRIC/qos.json.j2:224]: Good catch. Fixed. Thanks.

**Review by neethajohn — APPROVED** [review]

**Pull request merged**